onnxruntime/python/session.py

```diff
@@ -191,21 +191,57 @@ def _load_model(self, providers):
 
 
 class IOBinding:
+'''This class provides API to bind input/output to a specified device, e.g. GPU'''
     def __init__(self, session):
         self._iobinding = C.SessionIOBinding(session._sess)
 
+    def bind_input(self, name, numpy_arr_on_cpu):
```

Comment:
Let's explicitly mention here that the default device used to bound the input is CPU. Same comment for bind_output when all the device specific and shape params are not supplied by the user.

Reply:
We should also add a test case for this new function.

Reply:
Done.